Add documentation for `disableLineBreaks` property of `RichText` (#56284)
WordPress/gutenberg: packages/block-editor/src/components/rich-text/README.md

_Optional._ Placeholder text to show when the field is empty, similar to the
[`input` and `textarea` attribute of the same name](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/HTML5_updates#The_placeholder_attribute).

### `disableLineBreaks: Boolean`

> _Optional._ `Text` won't insert line breaks on `Enter` if set to `true`.

Reviewer: ```suggestion
_Optional._  Disables inserting line breaks on `Enter` when it is set to `true`
```
